server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -654,6 +677,7 @@ static final class Fields {
         static final String TIME = "time";
         static final String CURRENT = "current";
         static final String TOTAL = "total";
+        static final String SEARCH_IDLE_WAKEN_UP_TOTAL = "search_idle_waken_up_total";
```

Comment:
Should be `search_idle_waken_up_count_total` or if we're renaming then - `search_idle_retry_count_total`/`search_idle_reactivate_count_total`

Reply:
We increment 1 for each listener, and this test contains two listeners, so we increase twice. `final List<SearchOperationListener> indexingOperationListeners = new ArrayList<>(Arrays.asList(listener, listener));`

Reply:
Just realized that I replied at the wrong place..